static/styles/help/help.scss

```diff
@@ -27,12 +27,12 @@ label.form-group {
 }
 
 // Label
-.control-label {
-  &.headline{
-    font-weight: bold;
-    text-align: left;
-  }
-}
+  .control-label {
+    &.headline{
+      font-weight: bold;
+      text-align: left;
+    }
+  }  
```

Comment:
Why the additional indentations? This section seems to be "out of sync" with the rest of the file now.